src/js/settings.ts

```diff
@@ -151,7 +151,8 @@ export default {
     flipPieces: localstorageprop('settings.otb.flipPieces', false),
     useSymmetric: localstorageprop('settings.otb.useSymmetric', false),
     variant: localstorageprop('settings.otb.variant', 'standard'),
-    availableVariants: offlineAvailableVariants
+    availableVariants: offlineAvailableVariants,
+    savedGames: localstorageprop('settings.otb.savedGames', {})
```

Comment:
@veloce My idea here was to create a general data structure that we could use in the future if we decided to save all games (at which point the object keys would be gameIds), but to just hardcode it to 1 for now for the exactly the reason you described.

I'm happy to just turn this into a string holding the single pgn though. We can change later if we start saving games.